### PR TITLE
Fixes/transfer currency form

### PR DESCRIPTION
**Pull request by arpadt**
https://bullioncapital.atlassian.net/browse/ABX-4466

Fixes for the transfer form as per comments in https://github.com/KinesisNetwork/wallet-desktop/pull/70

### CODE REVIEW

File: src/services/transfer.ts [hunk 1]
```diff
@@ -99,3 +99,21 @@ async function newCreateAccountTransaction(
...
+    case ImageSize.large:
+      return { image: 'is-128x128', font: 'is-size-1' }
+      break
+    case ImageSize.medium:
+    default:
+      return { image: 'is-64x64', font: 'is-size-3' }
+  }
+}
+
+export const validateAmount = (amount: string) => {
```
Comment (desnor) [inline src/services/transfer.ts]: these are pretty generic helpers, maybe they'd fit better in the `services/utils` module? there is already one called `renderAmount` there

File: src/containers/TransferCurrency/TransferDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        label="Description"
+        placeholder="Optional"
+        helpText={`${props.memo.length || 0} / 25`}
+        errorText={props.errors.memo}
+      />
+    </React.Fragment>
+  )
+}
+
+const ConnectedTransferDetails = connect(
```
Comment (desnor) [inline src/containers/TransferCurrency/TransferDetails.tsx]: Not a biggie, but I feel like we could take the `Connected` part out of the name, instead just use `TransferFormDetails` which will keep the name a bit more concise and save the `as` call too. We've already got the `TransferFormDetailsPresentation` to differentiate

File: src/components/TransactionOperation.tsx [hunk 3]
```diff
@@ -40,11 +40,15 @@ const getAmount = (t: TransactionOperationView) => {
       amount = Number(t.operation.amount)
       break
     default:
-      return 'Other'
+      return amount
   }
   return t.isIncoming ? amount.toFixed(5) : (amount + Number(t.fee)).toFixed(5)
 }
 
+const isCurrencyOperation = (amount: number | string, currency: Currency) => {
```
[the remark this answers is not in the document]
Comment (arpadt) [reply src/components/TransactionOperation.tsx]: done

File: src/services/transfer.ts (hunk 1 above)
Comment (arpadt) [reply src/services/transfer.ts]: moved them

File: src/containers/TransferCurrency/TransferDetails.tsx (hunk 2 above)
Comment (arpadt) [reply src/containers/TransferCurrency/TransferDetails.tsx]: I followed @Carl-Foster 's naming practice but I changed it as you requested.

File: src/components/InitialsAvatar.tsx [hunk 4]
```diff
@@ -1,23 +1,21 @@
...
   size: string
-  fontSize: string
 }
 
 class InitialsAvatar extends React.Component<Props> {
   render() {
     return (
       <div
-        className={`image is-${this.props.size}x${this.props.size} has-background-grey is-flex`}
+        className={`image ${setImageSize(this.props.size).image} has-background-grey is-flex`}
```
[the remark this answers is not in the document]
Comment (desnor) [reply src/components/InitialsAvatar.tsx]: eg
```typescript
  const { font, image } = setImageSize(this.props.size)
```

[the remark this answers is not in the document]
Comment (arpadt) [reply src/components/InitialsAvatar.tsx]: done

**Pull request merged**